tests/data/README.md

```diff
@@ -0,0 +1,26 @@
+# Test Data Notes
+
+
+---
+
+## Databases
+
+`bsc5` database file sourced from:
```

Comment:
This database is brought in for test purposes? We otherwise use hip_main or tyc_main.

You might want to put in a comment along these lines.

Reply:
yeah, i chose that one because it is so small although anything under 10MB would probably be fine. When this is built as a wheel the test files don't get included so there are fewer concerns about large files.

Reply:
Let's add something like:

Note that BSC5, being limited to bright stars, is a small database which is convenient for testing. However, it is not suitable for building pattern databases with fields of view smaller than 20 degrees. Production users of Tetra3 should build their pattern databases from the 'hip_main' or 'tyc_main' catalogs.